ui/src/shared/components/TableGraphFormat.tsx

```diff
@@ -64,6 +65,25 @@ interface State {
   invalidDataError: ErrorTypes
 }
 
+const areFormatPropertiesEqual = (prevProperties, newProperties) => {
```

Comment:
is there a good type annotation for `prevProperties` or `newProperties`?

Reply:
i can add types